src/features/offer/components/SameArtistPlaylist/component/SameArtistPlaylist.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import { HitOfferWithArtistAndEan } from 'features/offer/components/SameArtistPlaylist/api/fetchOffersByArtist'
+import { PlaylistType } from 'features/offer/enums'
+import { Offer, RecommendationApiParams } from 'shared/offer/types'
+import { PassPlaylist } from 'ui/components/PassPlaylist'
+import { SectionWithDivider } from 'ui/components/SectionWithDivider'
+import { Spacer } from 'ui/theme'
+
+interface SameArtistPlaylistProps {
+  sameArtistPlaylist: HitOfferWithArtistAndEan[]
+  renderItem: (
+    props: {
+      item: Offer
+      width: number
+      height: number
+      playlistType?: PlaylistType
+    },
+    apiRecoParams?: RecommendationApiParams
+  ) => React.ReactElement
```

Comment:
Ton typage est mauvais. Je sais que tu as récupéré ça de `OfferBody.tsx` mais sur ce composant il y a un traitement de fait.
Là tu passes directement `renderItem` à `PassPlaylist` alors le vrai typage est
```suggestion
  renderItem: (
    props: {
      item: Offer
      width: number
      height: number
      playlistType?: PlaylistType
    }
  ) => React.ReactElement
```
Puisqu'il n'a aucun moyen de connaître les `apiRecoParams`